tests/integration_test/action_service_test.py

```
@@ -14523,8 +14813,8 @@ def test_prompt_action_response_action_crud_prompt_query_source_slot_value_strin
     ]
     assert resp_json['responses'][0]['text'] == generated_text
 
-@mock.patch.object(litellm, "aembedding", autospec=True)
-def test_prompt_action_response_action_crud_prompt_query_source_slot_app_exception(mock_embedding, aioresponses):
+
+def test_prompt_action_response_action_crud_prompt_query_source_slot_app_exception(aioresponses):
```

Ruff (0.14.14)

[error] Redefinition of unused `aioresponses` from line 7: `aioresponses` redefined here

(F811)